### PR TITLE
Create Setup Guide for wiki

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [x] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description
Add a Setup Guide page for the wiki to explain the first-time setup.

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

File: docs/Setup.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+The Profile-README can be created by creating a new Repository that matches your Username. For example, if your name is `User` would you name your Repository `User`.  
+You know you set it up when you get a message telling you that you discovered a secret.
+
+When creating the Repository, make sure you have setup the following things:
+
+- You set the Repository to `Public`
+- You checked `Add a README file`
+
+You can also use a template Repository such as the [`activity-template`][template] from Readme-Workflows to quickly setup your Profile-README. Important is only that the name of the repository is your own.
```
Comment (PuneetGopinath) [inline docs/Setup.md:19]:
```suggestion
You can also use a template Repository such as the [`activity-template`][template] provided by Readme-Workflows to quickly setup your Profile-README. Important is only that the name of the repository is your GitHub username.
```

File: docs/Setup.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+You're now ready to configure the file which we explain in the next step.
+
+### 1.1) Manual setup
+
+If you prefer a more manual setup can you create a file yourself manually.  
+To do that, make sure that you're on the `Code` tab of the Repository and on the root-level of it. You know you're right if you see info on the right-hand side such as `About`.
+
+From there click the `Add file` button followed by `Create new file`.  
+In the editor window should you now make sure that the file is saved in the right location. To do this, click the small textbox at the top and type `.github/workflows/<name>.yml` (`<name>` can be any name you want).
```
Comment (PuneetGopinath) [inline docs/Setup.md:81]:
```suggestion
In the editor window should you now make sure that the file is saved in the right location. To do this, click the small textbox at the top and type `.github/workflows/<name>.yml` (`<name>` can be any name you want, recommend to use `update-readme`).
```

File: docs/Setup.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+In the large text field will you put the contents of the workflow which we explain below.
+
+### 1.2) Configure file
+
+Now it's time to configure the workflow file.
+
+You can just copy the below text and replace any existing content in the editor with it.  
+Note that `{version}` should be replaced with the latest release of the Recent-Activity Action.
```
Comment (PuneetGopinath) [inline docs/Setup.md:90]:
```suggestion
Note that `{version}` should be replaced with the latest release of the Recent-Activity Action or use `main` to use the main branch.
```

File: docs/Setup.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: Readme-Workflows/recent-activity@{version}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+The action has some settings that are directly set through `with` options such as `CONFIG_FILE` which allows you to set a different config file to use (Explained next).
+
+Once you're donw with your setup commit the changes. You know you did it right if you can see `.github/workflows` in the directory where your README is located at.
```
Comment (PuneetGopinath) [inline docs/Setup.md:113]:
```suggestion
Once you're done with your setup commit the changes. You know you did it right if you can see `.github/workflows` in the directory where your README is located at.
```

File: docs/Setup.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+Now that you set up the workflow is it time to create and configure the YAML file for the Action.  
+By default is the file called `recent-activity.config.yml` and is located in the `.github` directory. If you set a different file-path through the `CONFIG_FILE` option in the workflow, make sure that the name and location match.
+
+When no configuration was defined would the action default to specific values:
+
+- `username` defaults to the Repository-Owner
+- `max_lines` defaults to 5
+- `readme_file` defaults to `./README.md`
+- `disabled_events` defaults to `comments`
```
Comment (PuneetGopinath) [inline docs/Setup.md:125]:
```suggestion
- `disabled_events` defaults to `["comments"]`
```

**Review by abhijoshi2k — APPROVED** [review]
Looks great! Thanks.

File: docs/Setup.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+Please see the [`sample.yml`][sample] file for all options and their default values.  
+You can also check the [[Configuration]] Page for any info about the different options.
+
+## 3) Configure README file
+
+The final step is to now update the README.md file. If you selected `Add a README file` should there be a README file in your Repository that you can just edit.
```
Comment (PuneetGopinath) [inline docs/Setup.md:132]:
```suggestion
The final step is to now update the README.md file. If you selected `Add a README file` when creating the repo. There be a README file in your Repository that you can just edit.  
```

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]
Looks good. Though I have requested some changes

File: docs/Setup.md (hunk 3 above)
Comment (abhijoshi2k) [reply docs/Setup.md:90]: `main` may have breaking changes. Using version is always safe!

Comment (PuneetGopinath) [reply docs/Setup.md:90]: Okay

File: docs/Setup.md (hunk 6 above)
Comment (abhijoshi2k) [reply docs/Setup.md:132]:
```
The final step is to now update the README.md file. If you selected `Add a README file` when creating the repo. There will be a README file in your Repository that you can just edit.  
```
@PuneetGopinath you missed the word `will`

Comment (PuneetGopinath) [reply docs/Setup.md:132]: @abhijoshi2k Directly edit my comment

File: docs/Setup.md (hunk 3 above)
Comment (Andre601) [reply docs/Setup.md:90]: You are aware that it says "should be replaced with the **latest release**" right?

File: docs/Setup.md (hunk 2 above)
Comment (Andre601) [reply docs/Setup.md:81]: Won't really add this... It doesn't matter what name is used here.

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**